op-service/signer/transaction_args.go

```diff
@@ -34,11 +34,11 @@ type TransactionArgs struct {
 	Input *hexutil.Bytes `json:"input"`
 
 	AccessList *types.AccessList `json:"accessList,omitempty"`
-	ChainID    *hexutil.Big      `json:"chainId,omitempty"`
+	ChainID    *hexutil.U256     `json:"chainId,omitempty"`
 
 	// Custom extension for EIP-4844 support
 	BlobVersionedHashes []common.Hash `json:"blobVersionedHashes,omitempty"`
-	BlobFeeCap          *hexutil.Big  `json:"maxFeePerBlobGas,omitempty"`
+	BlobFeeCap          *hexutil.U256 `json:"maxFeePerBlobGas,omitempty"`
 
 	// Custom extension for EIP-7702 support
 	AuthList []types.SetCodeAuthorization `json:"authList,omitempty"`
```

Comment:
```suggestion
	AuthList []types.SetCodeAuthorization `json:"authorizationList,omitempty"`
```

Geth and eth API spec for `eth_sendTransaction` use `authorizationList` as JSON name

Reply:
Nice catch